routes/items.js

```diff
@@ -1,27 +1,20 @@
 const express = require('express')
 const router = express.Router()
 
-router.get( '/', ( request, response, next ) => {
+router.get( '/', ( request, response ) => {
   const { user } = request
 
   response.render( 'items/index', { user })
 })
 
-router.post( '/create', ( request, response, next ) => {
-  // console.log( request.session )
-  // const user_id = request.session.passport.user.id
+router.post( '/items', ( request, response, next ) => {
```

Comment:
We can remove `next` here as well.  Also, this route should just be `/` (as it is currently, this would be `/items/items`...)